packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -0,0 +1,36 @@
+import { h } from '@stencil/core';
+import { getNodeName } from '../get-node-name.function';
+
+import { OzonContentNode } from '../ozon-content-node.interface';
+
+export class OzonContentFiguurNode implements OzonContentNode {
+  name = [
+    'Figuur',
+  ];
+
+  render(node: Element) {
+    const childNodes = Array.from(node.childNodes);
+    const titel = childNodes.find(n => getNodeName(n) === 'Titel')?.textContent;
+    const bijschrift = childNodes.find(n => getNodeName(n) === 'Bijschrift')?.textContent;
+
+    const illustratieNode = childNodes.find(n => getNodeName(n) === 'Illustratie');
+
+    if (illustratieNode instanceof Element) {
+      return (
+        <div class="dso-ozon-figuur">
+          <dso-image-overlay>
+            <img
+              src={illustratieNode.getAttribute('naam') ?? undefined}
+              alt={titel ?? illustratieNode.getAttribute('alt') ?? illustratieNode.getAttribute('naam') ?? undefined}
+              height={illustratieNode.getAttribute('hoogte') ?? undefined}
+              width={illustratieNode.getAttribute('breedte') ?? undefined}
+            />
+          </dso-image-overlay>
+          {bijschrift && (
+            <span class="figuur-bijschrift">{bijschrift}</span>
```

Comment:
Ik denk dat we hier nog een koppeling moeten leggen met de `<img>` waar dit bijschrift bij hoort.

Reply:
Ik heb https://github.com/dso-toolkit/dso-toolkit/issues/1730 aangemaakt om de volledige mapping van Figuur in kaart te brengen. De scope van dit issue is wat grijs, maar ik denk gezien de titel dat we ons vooral op de `dso-image-overlay` mapping moeten focusen.